run.sh

```diff
@@ -1,131 +1,40 @@
 #!/usr/bin/with-contenv bashio
 # shellcheck shell=bash
-
 bashio::log.info "Starting NoLongerEvil Add-on..."
 
-# Read user configuration from options
-DEBUG_LOGGING=$(bashio::config 'debug_logging')
-ENTRY_KEY_TTL_SECONDS=$(bashio::config 'entry_key_ttl_seconds')
-
-# Container always listens on these ports
-PROXY_PORT=8000
-CONTROL_PORT=8081
-INGRESS_PORT=8082
-
-# Get API_ORIGIN from user config (REQUIRED - must include port)
-if bashio::config.has_value 'api_origin'; then
-    API_ORIGIN=$(bashio::config 'api_origin')
-    bashio::log.info "Using configured API origin: ${API_ORIGIN}"
-
-    # Validate api_origin format: must be a valid URL with protocol and port
-    if ! echo "${API_ORIGIN}" | grep -qE '^https?://[^/:]+:[0-9]+$'; then
-        bashio::log.fatal "api_origin must be a valid URL with protocol, hostname, and port"
-        bashio::log.fatal "Current value: ${API_ORIGIN}"
-        bashio::log.fatal "Expected format: http://<hostname-or-ip>:<port>"
-        bashio::log.fatal "Examples:"
-        bashio::log.fatal "  http://192.168.1.100:9543"
-        bashio::log.fatal "  http://homeassistant.local:9543"
-        exit 1
-    fi
-else
-    bashio::log.fatal "api_origin is required! Please configure it in the add-on settings."
-    bashio::log.fatal "Example: http://192.168.1.100:9543"
-    exit 1
-fi
-
-# Check if MQTT service is available (REQUIRED)
-bashio::log.info "Checking for MQTT service..."
-if bashio::services "mqtt" "host" > /dev/null 2>&1; then
-    bashio::log.info "MQTT service IS available from Supervisor"
-    
-    # Extract MQTT credentials from Supervisor services API
-    MQTT_HOST=$(bashio::services "mqtt" "host")
-    export MQTT_HOST
-    MQTT_PORT=$(bashio::services "mqtt" "port")
-    export MQTT_PORT
-    MQTT_USER=$(bashio::services "mqtt" "username")
-    export MQTT_USER
-    MQTT_PASSWORD=$(bashio::services "mqtt" "password")
-    export MQTT_PASSWORD
-    
-    bashio::log.info "MQTT service configured:"
-    bashio::log.info "  Host: ${MQTT_HOST}"
-    bashio::log.info "  Port: ${MQTT_PORT}"
-    bashio::log.info "  User: ${MQTT_USER}"
+# Read options from Home Assistant
+OPTIONS_FILE="/data/options.json"
```

Comment:
I might ask to move the options from options.json and move it into `config.yaml` into the options and the schema 

Do you prefer options.json for a particular reason